### ISSUE TITLE
Ability to easily exclude characters from a StringArbitrary

### ISSUE THREAD

**Issue by osi** [jlink/jqwik#167]
We have a customization on the default `StringArbitrary` that excludes the null character, `\0`, as PostgreSQL does not allow null bytes inside of strings. 

With 1.3, this was implemented as

```
Arbitraries.strings()
                   .withChars(Arbitraries.chars()
                                         // Filter out a null character, as Postgresql won't store them.
                                         .filter(c -> c != 0));
```

That form broke with the upgrade to 1.4, and the new simplest form appears to be:

```
        Arbitraries.strings()
                   .withChars(Arbitraries.chars()
                                         // Filter out a null character, as Postgresql won't store them.
                                         .filter(c -> c != 0)
                                         .allValues()
                                         .orElseThrow()
                                         .map(Object::toString)
                                         .collect(Collectors.joining())
                   );
```

There's no problem with the new form, other than being more verbose. I don't know how common this usage pattern may be, so a valid resolution is to close the ticket as a means of capturing how to do this.

**Comment by jlink** [comment]
Your workaround is largely inefficient since it will create a large list of chars for each generator. That’s why I think that a native filtering mechanism for characters makes a lot of sense.

**Comment by jlink** [comment]
Functionality added in https://github.com/jlink/jqwik/commit/3ab3db2a083f5d15d4375df5b24e31f63c965079.

Is available in 1.5.1-SNAPSHOT
